sdk/maps/Azure.Maps.Render/tests/Samples/RenderClientSamples.cs

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Drawing;
+#region Snippet:SaveToFile
+using System.IO;
+#endregion
+#region Snippet:RenderImportNamespace
+using Azure.Maps.Render;
+#endregion
+using Azure.Core.GeoJson;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Maps.Render.Tests
+{
+    public class RenderClientSamples : SamplesBase<RenderClientTestEnvironment>
+    {
+        public void RenderClientViaAAD()
+        {
+            #region Snippet:InstantiateRenderClientViaAAD
+            // Create a MapsRenderClient that will authenticate through Active Directory
+            var credential = new DefaultAzureCredential();
+            var clientId = "<My Map Account Client Id>";
+            MapsRenderClient client = new MapsRenderClient(credential, clientId);
+            #endregion
+        }
+
+        public void RenderClientViaSubscriptionKey()
+        {
+            #region Snippet:InstantiateRenderClientViaSubscriptionKey
+            // Create a MapsRenderClient that will authenticate through Subscription Key (Shared key)
+            var credential = new AzureKeyCredential("<My Subscription Key>");
+            MapsRenderClient client = new MapsRenderClient(credential);
+            #endregion
+        }
+
+        [Test]
+        public void RenderingImageryTiles()
+        {
+            #region Snippet:RenderImageryTiles
+            var credential = new DefaultAzureCredential();
+            var clientId = TestEnvironment.MapAccountClientId;
+            var client = new MapsRenderClient(credential, clientId);
+
+            #region Snippet:GetTileXY
+            int zoom = 10, tileSize = 300;
+
+            // Get tile X, Y index by coordinate, zoom and tile size information
+            TileMath.PositionToTileXY(
+                new GeoPosition(13.3854, 52.517), zoom, tileSize,
+                out var tileX, out var tileY
+            );
+            #endregion
+
+            #region Snippet:RenderImagery
+            // Get imagery tile
+            var imageryTile = client.GetMapImageryTile(new TileIndex(tileX, tileY, zoom));
+            Assert.IsNotNull(imageryTile);
+
+            // Prepare a file stream to save the imagery
+            using (var fileStream = File.Create(".\\BerlinImagery.png"))
+            {
+                imageryTile.Value.CopyTo(fileStream);
+                Assert.IsTrue(fileStream.Length > 0);
+            }
+            #endregion
+            #endregion
```

Comment:
```suggestion
```

Reply:
I think this is a duplicate?   Maybe I'm misreading?

Reply:
It's a embedded `#region` here. I believe it's correct here :)